scripts/java-api-check.json

```diff
@@ -52,7 +52,8 @@
             "com\\.mapbox\\.maps\\.plugin\\.locationcomponent\\.ModelLayerRenderer",
             "com\\.mapbox\\.maps\\.plugin\\.locationcomponent\\.ModelLayerWrapper",
             "com\\.mapbox\\.maps\\.plugin\\.locationcomponent\\.ModelSourceWrapper",
-            "com\\.mapbox\\.maps\\.plugin\\.scalebar\\.RefreshHandler",
+            "com\\.mapbox\\.maps\\.plugin\\.scalebar\\.ScaleBarImpl",
+            "com\\.mapbox\\.maps\\.plugin\\.scalebar\\.ScaleBarImpl$RefreshHandler",
```

Comment:
@Chaoba My understanding is that previously the `RefreshHandler` was not picked up properly since it was a `ScaleBarImpl` inner class.